rdr_service/offline/biobank_samples_pipeline.py

```diff
@@ -145,7 +146,7 @@ def get_last_biobank_sample_file_info(monthly=False):
     if monthly:
```

Comment:
Do we still need this logic if we're screening on specific file name formats now?

Reply:
Good catch. I don't think we'd need it, but it could help with narrowing things down when searching through names. I don't see any immediate need to take it out.